Flow.Launcher.Infrastructure/Logger/Log.cs

```diff
@@ -23,23 +28,44 @@ static Log()
             }
 
             var configuration = new LoggingConfiguration();
-            var target = new FileTarget();
-            configuration.AddTarget("file", target);
-            target.FileName = CurrentLogDirectory.Replace(@"\", "/") + "/${shortdate}.txt";
+
+            const string layout = 
+                @"${date:format=HH\:mm\:ss.ffffK} - " +
+                @"${level:uppercase=true:padding=-5} - ${logger} - ${message:l}" +
+                @"${onexception:${newline}" +
+                    @"EXCEPTION OCCURS\: ${exception:format=tostring}${newline}}";
+            
+            var fileTarget = new FileTarget
+            {
+                FileName = CurrentLogDirectory.Replace(@"\", "/") + "/${shortdate}.txt",
+                Layout = layout
+            };
+
+            var fileTargetASyncWrapper = new AsyncTargetWrapper(fileTarget);
+
+            var debugTarget = new OutputDebugStringTarget
+            {
+                Layout = layout
+            };
```

Comment:
The target to output information to debug console (not the console target for command line app).